modules/hp_graph/templates/paragraph--hp-graph.html.twig

```diff
@@ -0,0 +1,27 @@
+{#
+/**
+ * @file
+ * HP Graph paragraph.
+ *
+ * @ingroup themeable
+ */
+#}
+
+<section class="hp_graph_bundle" >
+	<div class="featurettes_text col">
+		<aside class="featurette">
+			<h1 class="featurette-heading">{{-content.field_hp_graph_title[0]-}}</h1>
+			<p class="lead">{{-content.field_hp_graph_description[0]-}}</p>
+			{%  if true in content.field_hp_graph_link[0] %}
+				<a class="btn" href="{% spaceless %}{{-content.field_hp_graph_link[0]['#url']-}}{% endspaceless %}">{{-content.field_hp_graph_link[0]['#title']-}}</a>
+			{% endif %}
+		</aside>
+	</div>
+	<div class="featurettes_image col">
+		{% for i in 0..content.field_hp_graph_data|length %}
```

Comment:
should this indeed be "0..content" or "0.content"?

Reply:
Yes this should be "0..content